### PR TITLE
Add memory usage chart

### PR DESCRIPTION
**Pull request by rerobika**
Added memory usage chart which dinamically changing based on the recieved data from debugger. Every breakpoint updates the chart with actual memory usage data. Changed the order of panels for more confortable use.

### CODE REVIEW

**Comment by rerobika** [comment]
Modified!

**Comment by rerobika** [comment]
Fixed resize bug!

File: index.html [hunk 1]
```diff
@@ -23,13 +23,24 @@
   <!-- Custom CSS -->
   <link href="css/custom.css" rel="stylesheet">
 
+  <!-- Chart CSS -->
```
Comment (knightburton) [inline index.html]: Please move this up over the custom stylesheet.

File: index.html [hunk 2]
```diff
@@ -23,13 +23,24 @@
...
+  <link rel="stylesheet" type="text/css" href="./css/Chart/c3.css">
+
   <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
   <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
   <!--[if lt IE 9]>
   <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
   <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
   <![endif]-->
 
+  <!-- Chart source -->
```
Comment (knightburton) [inline index.html]: Please move these JS links down below to the end of the body, and remove the `./` from the paths.

File: js/jerry-client.js [hunk 3]
```diff
@@ -73,6 +73,7 @@ var keybindings = {
   custom : null, // Create own bindings here.
 };
 
+var breakpointInformationToChart = "";
```
Comment (knightburton) [inline js/jerry-client.js]: Move this variable into the session class please.

File: js/jerry-client.js [hunk 4]
```diff
@@ -2095,6 +2121,9 @@ function DebuggerClient(address)
         }
         /* EXTENDED CODE */
 
+        /*Add breakpoint information to chart*/
```
Comment (knightburton) [inline js/jerry-client.js]: Move this code snippet into the extended code comment please.

**Comment by rerobika** [comment]
Fixed style issues!

**Comment by rerobika** [comment]
Fixed js files positions!

**Comment by knightburton** [comment]
Awesome! :)

**Pull request merged**